### DISCUSSION TITLE
Mattermost/Slack integration?

### DISCUSSION THREAD

**Discussion by gnacoding** [toeverything/AFFiNE#586] [Ideas]
Hi All!

Please excuse if this has been already answered (couldn't find any info). 
Are there any plans to create an integration so you can search for documents on affine by using mattermost for example?

I'm thinking about something like outline has (https://www.getoutline.com/)

**Comment by c-nv-s** [comment]
I was also looking to evaluate between Outline and Affine and couldn't find any info on this integration. anyone know if one exists?
